python/src/FunctionalChaosResult_doc.i.in

```diff
@@ -159,4 +170,122 @@ basis : list of :class:`~openturns.Function`
 Returns
 -------
 transformation : :class:`~openturns.Function`
-    Transformation :math:`T` such that :math:`T(\vect{X}) = \vect{Z}`."
+    Transformation :math:`T` such that :math:`T(\physicalInputRandomVector) = \standardInputRandomVector`."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::FunctionalChaosResult::getConditionalExpectation
+"Get the conditional expectation of the expansion given one vector input.
+
+This method returns the functional chaos result corresponding to the conditional 
+expectation of the output given an input vector.
+Indeed, the conditional expectation of a polynomial chaos expansion is, again, 
+a polynomial chaos expansion.
+Let :math:`\physicalInputRandomVector \in \Rset^{\physicalInputDimension}` be the input and let :math:`\vect{u} \subseteq \{1, ..., \physicalInputDimension\}`
+be a set of marginal indices. 
+Let :math:`\physicalInputRandomVector_{\vect{u}} \in \Rset^{|\vect{u}|}` be the vector 
+corresponding to the group of input variables where :math:`|\vect{u}| = \operatorname{card}(\vect{u})`
+is the number of input variables in the group.
+This function returns the functional chaos expansion of:
+
+.. math::
+
+    \widehat{g}\left(\physicalInputObservation_{\vect{u}}\right) 
+    = \Expect{Y | \physicalInputRandomVector_{\vect{u}} = \physicalInputObservation_{\vect{u}}}
+
+for any :math:`\physicalInputObservation_{\vect{u}} \in \Rset^{|\vect{u}|}`.
+
+*Mathematical analysis*
```

Comment:
use ** for bold, this is too thin

Reply:
Fixed at a7a2d782211ea61409b9a0f85cb552473872a914